src/main/java/com/lucidworks/zeppelin/solr/SolrInterpreter.java

```diff
@@ -254,4 +256,34 @@ public SolrClient getCloudClient() {
   public InterpreterResult returnCollectionNull() {
       return new InterpreterResult(InterpreterResult.Code.INCOMPLETE, InterpreterResult.Type.TEXT, "Set collection to use with 'use {collection}' command or set collection in query params for search and facet commands");
   }
+
+  // Only called when args[0].equals("help")
+  private InterpreterResult determineHelpResponse(String[] args) {
+    String msg = null;
+    if (args.length == 1) {
+      msg = "List of supported commands: " + COMMAND_LIST_STRING + ". Run `help <command>` for information on a specific command";
+    } else if (args.length == 2) {
+      switch (args[1]){
+        case "use":
+          msg = "Set a default collection for us in other commands.\\nUsage: `use <collection_name>`";
+          break;
+        case "search":
+          msg = "Issue a search request and display the results.\\nUsage: `search <Solr query params>`";
+          break;
+        case "facet":
+          msg = "Issue a facet request and display the computed counts.\\n Usage: `facet <Solr facet params>`";
+          break;
+        case "stream":
+          msg = "Issue a streaming expression request and display the results.\\nUsage: `stream <streaming-expression>`";
+          break;
+        case "sql":
+          msg = "Issue a SolrSQL query and display the results\\nUsage: `sql <sql-expression>`";
```

Comment:
Is "SolrSQL" really a concept?   Maybe just `Issue a sql query to solr and display the results`, or maybe, `Issue a streaming sql query and display the results`?    Less certain on the right words to use!

Reply:
Good question.  I swear I've heard it used to refer to the subset of SQL syntax that Solr supports.  That's what I was trying to get across there: that Solr only supports a small subset of SQL, so don't expect any old SQL to work here.

Googling for whether that's used though doesn't turn up much.  There's at least [one place](https://lucene.apache.org/solr/guide/8_6/parallel-sql-interface.html#solr-sql-syntax) where that's used.  But for all I know that's a typo.  It's def not common in any case.  I'll think about the wording a bit.